src/SDKs/TrafficManager/TrafficManager.Tests/TrafficManager.Tests.csproj

```diff
@@ -10,7 +10,7 @@
     <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.6.0-preview" />
```

Comment:
Please confirm tests can run without this package reference.

Reply:
Yes, they work fine. We are not using this package anymore. I've already made required changes in test code.